GeoInterfaceMakie/src/GeoInterfaceMakie.jl

```diff
@@ -126,6 +144,24 @@ end
 @enable GeoInterface.Wrappers.WrapperGeometry
 
 
+# Creating empty geometries from traits
+function _geomtrait_for_array(arr)
+    idx = findfirst(!ismissing, arr)
+    geom = if isnothing(idx)
+        error("We can't plot only missings!!")
+    else
+        arr[idx]
+    end
+end
+
+# NaN geometry creators from traits
+_nan_geom(::GI.PointTrait, ndims = 2, T = Float64) = GB.Point{ndims, T}(NaN)
+_nan_geom(::GI.MultiPointTrait, ndims = 2, T = Float64) = GB.MultiPoint(_nan_geom(GI.PointTrait(), ndims, T))
```

Comment:
This can't just be empty?

Reply:
Yeah I meant just an empty vector typed as NaN points

Reply:
That would work for poly but not lines or scatter...